sources/ReplicantWindow.cpp

```diff
@@ -7,16 +7,20 @@
  */
 
 
+#include <Catalog.h>
 #include <ControlLook.h>
 #include <LayoutBuilder.h>
 
 #include "DropZoneTab.h"
 #include "ReplicantWindow.h"
 
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "ReplicantWindow"
+
 
 ReplicantWindow::ReplicantWindow(BRect frame)
 	:
-	BWindow(BRect(0, 0, 120, 100), ("Resize"),
+	BWindow(BRect(0, 0, 120, 100), (B_TRANSLATE("Resize")),
```

Comment:
I wondered what "Resize" meant. Changing it to "Size" would be more confusing, I think. Maybe a titleless window with `SetMinimumSize()`?